### ISSUE TITLE
Response contains my API key

### ISSUE THREAD

**Issue by melissamcewen** [Airtable/airtable.js#82]
Right now when I do:
```
base('mybase').select({
    // Selecting the first 3 records in Grid view:
    maxRecords: 3,
    view: "Grid view"
}).eachPage(function page(records, fetchNextPage) {
    // This function (`page`) will get called for each page of records.

    records.forEach(function(record) {
        console.log('Retrieved', record.get('Ingredient'));
    });

    // To fetch the next page of records, call `fetchNextPage`.
    // If there are more records, `page` will get called again.
    // If there are no more records, `done` will get called.
    fetchNextPage();

}, function done(err) {
    if (err) { console.error(err); return; }
});

```

The response (that's console logged here) contains something that looks like this:
```
{
      "_table": {
        "_base": {
          "_airtable": {
            "_apiKey": "myrealapikey",
            "_endpointUrl": "https://api.airtable.com",
            "_apiVersion": "0.1.0",
            "_apiVersionMajor": "0",
            "_allowUnauthorizedSsl": false,
            "_noRetryIfRateLimited": false,
            "requestTimeout": 300000
          },
          "_id": "something"
        },
        "id": null,
        "name": "something"
      },
```

This causes some issues when I'm working on for example doing live coding or demo projects (I work in developer education). This happened to a colleague of mine and he had to reset his key.

**Comment by EvanHahn** [comment]
That's surprising! Is that happening from this line?

```javascript
console.log('Retrieved', record.get('Ingredient'));
```

Or another one?

**Comment by melissamcewen** [comment]
Yes, if I console log the whole record. Here's a working example. If you could see the console logs you could see my API key https://glitch.com/edit/#!/scaretable-page-request?path=server.js:43:51

**Comment by melissamcewen** [comment]
ahh it's not the server respond, it's just the way the SDK is formatting the stuff for me to use.

**Comment by syrnick** [comment]
Correct, that's not in the server response, but rather in-memory representation on the client. 

That said, it is fixable. I.e. we can probably fix it so that accidentally logging any objects returned by the SDK does not reveal the API key.

**Comment by melissamcewen** [comment]
Ah cool, yeah that would be a great service for us using the API to teach and livestream.

**Comment by EvanHahn** [comment]
We believe we've fixed this in #83 and will deploy a new version of Airtable.js soon!

**Comment by EvanHahn** [comment]
A fix for this has been published in `airtable@0.5.10`. Let us know if this helps!
